csharp/bankocr.Tests/TestGuest.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using bankocr;
+using Xunit;
+
+namespace TestGuest.Tests
+{
+    public class TestGuest
+    {
+        public TestGuest()
+        {
+
+        }
+        [Fact]
+        public void WhenThereAreManyMenThanWomen()
+        {
+            Guest objectguest = new Guest();
+            int [] guests = {-1, -1, -1, -1, 1, 1, 1, 1, 1, 1};
+            var result = objectguest.ReturnInviteMoreWomen(guests);
+            var expected = true;
+            Assert.Equal(expected, result);
+        }
+        
+        [Fact]
+        public void WhenThereAreManyWomenThanMen()
+        {
+            Guest objectguest = new Guest();
```

Comment:
Maybe "Guest" will not know how to say when to invite more women, we need to improve this class name